README.md

```diff
@@ -45,18 +66,28 @@ hashi-helper [--global-flags] command [--command-flags]
 - `--config-file` / `CONFIG_FILE`: A single `hcl` configuration file to parse instead of a directory (optional; default: `<empty>`)
 - `--environment` / `ENVIRONMENT`: The environment to process for (optional; default: `all`)
 - `--application` / `APPLICATION`: The application to process for (optional; default: `all`)
+- `--warn-unencrypted` / `WARN_UNENCRYPTED`: Issue a warning if unencrypted HCL files are discovered.
+- `--no-keybase-team` / `NO_KEYBASE_TEAM`: Ignore the Keybase team name in `.hashi-helper-config.yml.pgp` (if set).
```

Comment:
maybe make this into `KEYBASE_TEAM`, and if the ENV, CLI flag or the YAML value is empty, then behave like `--no-keybase-team` ?